src/object-list.scss

```diff
@@ -0,0 +1,212 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./list";
+
+$block: #{$fd-namespace}-object-list;
+
+.#{$block} {
+
+  $fd-object-list-with-value-zero: 0;
+  $fd-object-list-with-padding-value-half: 0.5rem;
+  $fd-object-list-with-padding-value-one: 1rem;
+  $fd-object-list-with-value-half: 0.75rem;
+  $fd-object-list-align-left: left;
+  $fd-object-list-align-right: right;
+  $fd-object-list-font-color: var(--sapContent_LabelColor);
+
+  &__item {
+    @include padding-top-bottom(1rem);
+
+    height: 100%;
+
+    .#{$fd-namespace}-list__link {
+      @include fd-active() {
+        .#{$block}__intro,
+        .#{$block}__object-attribute,
+        .#{$fd-namespace}-avatar,
+        .#{$fd-namespace}-object-identifier__title,
+        .#{$fd-namespace}-object-marker__text,
+        .#{$fd-namespace}-object-marker__icon,
+        .#{$fd-namespace}-object-number__text,
+        .#{$fd-namespace}-object-number__unit,
+        .#{$fd-namespace}-object-status__text,
+        .#{$fd-namespace}-object-status__icon {
+          color: var(--sapList_Active_TextColor);
+          text-shadow: none;
```

Comment:
@jacobdevera Is it ok to remove the text shadow here? I thought we decided it should stay for a11y reasons.

Reply:
Testers have raised concerns to handle text shadow that's why I made it as none.

 I removed it for now and will inform them to post here in case of concerns on this hope that will help.